core/players-handbook-2024/classes/class-cleric.xml

```diff
@@ -0,0 +1,447 @@
+		</sheet>
+		<rules>
+			<stat name= "divinespark:dice" value="1" level="2" />
+			<stat name= "divinespark:dice" value="1" level="7" />
+			<stat name= "divinespark:dice" value="1" level="13" />
+			<stat name= "divinespark:dice" value="1" level="18" />
+		</rules>
+	</element>
+
+	<element name="Level 3: Cleric Subclass" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_CLERICSUBCLASS">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_CLERIC_DIVINE_DOMAIN</requirements>
+		<description>
+			<p>You gain a Cleric subclass of your choice. The Life Domain, Light Domain, Trickery Domain, and War Domain subclasses are detailed after this class's description . A subclass is a specialization that grants you features at certain Cleric levels. For the rest of your career, you gain each of your subclass's features that are of your Cleric level or lower.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<select type="Archetype" name="Divine Domain" supports="Divine Domain" />
+		</rules>
+	</element>
+
+	<element name="Level 4: Ability Score Improvement" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_ABILITYSCOREIMPROVEMENT_CLERIC">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_CLERIC_ABILITY_SCORE_IMPROVEMENT</requirements>
+		<compendium display="false" />
+		<description>
+			<p>When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2, or you can increase two ability scores of your choice by 1. As normal, you can’t increase an ability score above 20 using this feature.</p>
+		</description>
+		<sheet display="false" />
+		<rules>
+			<!-- Verify & correct 2024 ASI implementation
+			<select type="Class Feature" name="Improvement Option (Cleric 4)" supports="Improvement Option,Cleric,4" level="4" />
+			<select type="Class Feature" name="Improvement Option (Cleric 8)" supports="Improvement Option,Cleric,8" level="8" />
+			<select type="Class Feature" name="Improvement Option (Cleric 12)" supports="Improvement Option,Cleric,12" level="12" />
+			<select type="Class Feature" name="Improvement Option (Cleric 16)" supports="Improvement Option,Cleric,16" level="16" /> -->
+		</rules>
+	</element>
+
+	<element name="Level 7: Blessed Strikes" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_CLERIC_BLESSEDSTRIKES">
+		<description>
+			<p>Divine power infuses you in battle. You gain one of the following options of your choice (if you get either option from a Cleric subclass in an older book, use only the option you choose for this feature);</p>
+			<p class="indent"><i>Divine Strike</i> or <i>Potent Spellcasting</i></p>
```

Comment:
```suggestion
			<p class="indent"><em><strong>Divine Strike.</strong></em> Once on each of your turns when you hit a creature with an attack roll using a weapon, you can cause the target to take an extra ld8 Necrotic or Radiant damage (your choice).</p>
			<p class="indent"><em><strong>Potent Spellcasting.</strong></em> Add your Wisdom modifier to the damage you deal with any Cleric cantrip.</p>

```

Reply:
See Divine Order comments for similar logic